hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/portable/PortableTest.java

```diff
@@ -728,7 +746,7 @@ public void testWriteObjectWithCustomSerializable() {
                 .setTypeClass(CustomSerializationTest.Foo.class);
         config.addSerializerConfig(sc);
         SerializationService serializationService = new DefaultSerializationServiceBuilder().setPortableVersion(1)
-                .addPortableFactory(PORTABLE_FACTORY_ID, new TestPortableFactory()).setConfig(config).build();
+                                                                                            .addPortableFactory(PORTABLE_FACTORY_ID, new TestPortableFactory()).setConfig(config).build();
```

Comment:
nit: this line is too long after the update. Could be improved by moving `.setPortableVersion(1)` into a new line